src/__tests__/quotes-test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import Routes from '../components/routes';
+
+import Quotes from '../components/quotes';
+
+describe('Quotes', () => {
+  test('quotes component is rendered correctly', () => {
+    const component = render(
+      <Routes><Quotes /></Routes>,
+    );
+    expect(component).toMatchSnapshot();
+  });
+
+  test('Text match Home page test', () => {
+    const { getByText } = render(<Routes><Quotes /></Routes>);
+    const text = getByText('Math-magician');
+    // eslint-disable-next-line no-unused-expressions
+    expect(text).toBeInTheDocument;
+  });
+});
```

Comment:
- [ ] Kindly enable Eslint above and find a workaround fixing the lint error, it is meant as a guide to help you write clean and professional code following the programming standards.

Reply:
Thanks for the suggestion. I had to import jest-dom from @testing